### PR TITLE
Corner radius support

### PR DESCRIPTION
**Pull request by jogboms**

### CODE REVIEW

**Comment by jogboms** [comment]
Fixes #5

File: src/gradient-common.ts [hunk 1]
```diff
@@ -7,6 +7,11 @@ const directionProperty = new Property<Gradient, string>({
   defaultValue: "to bottom"
 });
 
+const radiusProperty = new Property<Gradient, string>({
+  name: "radius",
+  defaultValue: 0
```
Comment (EddyVerbruggen) [inline src/gradient-common.ts]: On line 10 the property is defined as type `string`, so TSC doesn't like this default - but no worries, I'll merge and adapt it.

**Comment by EddyVerbruggen** [comment]
AWESOME! I'll merge and test now 👍 👍

File: src/gradient-common.ts (hunk 1 above)
Comment (jogboms) [inline src/gradient-common.ts]: Oh my! I totally missed that. 👍

**Pull request merged**